add-on/preferences/prefs.js

```diff
@@ -33,11 +33,7 @@ pref("loop.CSP", "default-src 'self' about: file: chrome:; img-src * data:; font
 pref("loop.fxa_oauth.tokendata", "");
 pref("loop.fxa_oauth.profile", "");
 pref("loop.support_url", "https://support.mozilla.org/1/firefox/%VERSION%/%OS%/%LOCALE%/cobrowsing");
-#ifdef LOOP_BETA
 pref("loop.facebook.enabled", true);
```

Comment:
It is checked in conversation.jsx (look for `facebook.enabled`), and then passed through to `DesktopRoomInvitationView`.

Its still a little new and hasn't been fully released yet, so I was going on the conservative side to keep it in for a bit longer yet, in case it needed backing out for any reason.